main/webapp/modules/core/scripts/facets/list-facet.js

```diff
@@ -161,7 +161,7 @@ ListFacet.prototype._initializeUI = function() {
           '<a href="javascript:{}" class="facet-choice-link" bind="resetButton">'+$.i18n('core-facets/reset')+'</a>' +
           '<a href="javascript:{}" class="facet-choice-link" bind="invertButton">'+$.i18n('core-facets/invert')+'</a>' +
           '<a href="javascript:{}" class="facet-choice-link" bind="changeButton">'+$.i18n('core-facets/change')+'</a>' +
-          '<span bind="titleSpan"></span>' +
+          '<span class="facet-title-span" bind="titleSpan" title="'+$.i18n('core-facets/edit-facet-title')+'"></span>' +
```

Comment:
I see this was done in the same way above (so it makes sense to imitate it for consistency), but just for the record, this is really bad: the HTML code can be broken easily by a translation which would contain certain characters. We should rather apply the title via JS DOM manipulation, which gets rid of escaping issues (just like this is done in most other places).

Reply:
Yes, I usually try to follow the coding style present, but I understand in the future you would rather always have JS DOM manipulation, and I agree. So I did it, and 10 other also.